lib/remote.provider.js

```diff
@@ -1,7 +1,113 @@
 'use strict';
 
+const jstp = require('@metarhia/jstp');
+
 const { StorageProvider } = require('./provider');
+const { RemoteCursor } = require('./remote.cursor');
+
+const APP_NAME = 'GLOBAL_STORAGE';
+const INTERFACES = ['provider', 'cursor'];
+
+class RemoteProvider extends StorageProvider {
+  // Create RemoteProvider
+  // cursor - RemoteCursor instance
+  // apiPromise - jstp promise
+  constructor() {
+    super();
+    this.api = null;
+    this.connection = null;
+    this.cursors = [];
+  }
+
+  // Open StorageProvider
+  //   callback - function(err, RemoteProvider)
+  // eslint-disable-next-line no-unused-vars
+  open(options, callback) {
+    jstp.ws.connectAndInspect(
+      APP_NAME,
+      null,
+      INTERFACES,
+      options,
```

Comment:
This makes it impossible to pass more than one option to the `connectAndInspect()` method, there should be an array of arguments that will be passed to this method using the spread operator.